cirrus/server/cirrus/main.py

```diff
@@ -76,9 +82,11 @@ def read_root():
 
 
 @app.post("/v1/features/", status_code=status.HTTP_200_OK)
-async def compute_features():
-    # # will recieve as part of incoming request
-    targeting_context: Dict[str, Any] = {"clientId": "test", "requestContext": {}}
+async def compute_features(request_data: FeatureRequest):
+    targeting_context: Dict[str, Any] = {
```

Comment:
Could we not reuse the type structure we declared above here?

Reply:
request data expect` client_id`, sdk need `clientId`, I can either change the request parameter key and then use this as a type too, or make another structure for this, what do you think?

Reply:
Oh I see.  Oh well anyways, we know we need to at some point declare actual types for all these dicts, including other ones in other parts of the code.  I guess we can do that later as part of hardening.